www/src/views/starter-showcase/showcase-list.js

```diff
@@ -143,9 +136,9 @@ const ShowcaseList = ({ urlState, items, imgs, count, sortRecent }) => {
                   </span>
                 </div>
                 <div>
-                  <span className="title">
+                  <span>
                     <h5 css={{ margin: 0 }}>
-                      <strong>{repo.name}</strong>
+                      <strong className="title">{repo.name}</strong>
```

Comment:
👍

Reply:
Now we only need to make the starter title clickable/link to the detail page ;-) (like the [site showcase does](https://next.gatsbyjs.org/showcase/)).

Actually I cursed myself out numerous times for designing the starter showcase cards with the GitHub username sitting _above_ the site title. Didn't think much of how much that would complicate things (not saying it's not doable).

An easy way around this would be to move the author's GitHub username and the "icon links" below the starter title. What do you think of that @KyleAMathews?

Reply:
or like this?

![screen shot 2018-09-09 at 01 07 23](https://user-images.githubusercontent.com/36336875/45259426-c6836400-b3cc-11e8-8972-05978da8b4ac.png)

Reply:
Initial reaction was a clear "yes", then I started wondering about the title wrapping, truncation etc.—how does that hold up to the rest of the starters? Looking good across breakpoints?

Reply:
you're right, doesn't always work

👇?
![screen shot 2018-09-09 at 01 43 25](https://user-images.githubusercontent.com/36336875/45259650-c6399780-b3d1-11e8-9afc-31c39e080b86.png)

Reply:
Haha nice! Welcome to my world of pain! 😉 

Honestly, seeing https://github.com/gatsbyjs/gatsby/pull/7998#discussion_r216142879 makes looking at this a bit hard for me now! ;)

Reply:
what do you mean it makes it hard?

Reply:
Oh sorry for not being clear about this! I meant to say I like https://github.com/gatsbyjs/gatsby/pull/7998#discussion_r216142879 so much that going back to four rows of title-and-meta is a bit hard now. 😄 

Also wondering again if we really, really need the "copy to clipboard" button there—I don't think so TBH (just checked and it also wasn't in my [mocks](https://www.figma.com/file/G3x7aYuyigYaibEpyxMsKLcP/gatsbyjs.org%3Ashowcase-starters?node-id=0%3A62)), and if the "Visit demo" link here is something we should drop here, or replicate for the site showcase :)

Reply:
Hey @riencoertjens, I'm so sorry I didn't have the time to properly respond to this today! Now it's a little late and I hope I can still manage to make my answer match the effort you put into this…let me try!

The two screenshots you posted look great, thank you so much for actively participating in this! ❤️🙏

While I'm still not 100% happy with the distribution of all the meta information of the starter showcase cards (but have no better answer right now either), I vote 👍for getting this in as it

* solves the problem of starter titles currently not linking to the detail page
* adds the Gatsby version info
* removes the IMO not needed "copy to clipboard" button on the starter showcase cards
* with adding the "link to demo" button to the site showcase cards, and updating their position for the starter showcase cards, streamlines the "action icon buttons" (link to demo/site, GitHub source) across both card variants

Pinging @shannonbux and @KyleAMathews for opinions 🙏

Reply:
Hi @fk and @riencoertjens! Thanks for doing this @riencoertjens and here is my feedback on the proposed design changes:

- I think all the proposed design changes are excellent.
- Only piece of feedback is use issue #7785 for the Gatsby v2 or Gatsby v1 buttons (we want them to be grey and neutral, and eventually we'll set a default filter for all v2 sites to gently encourage people to upgrade their starters to v2).
- Let's track these proposed design changes in a new issue! Would you mind creating that?

Also, if you're curious to see the overall starter showcase progress, here are the EPICS (you have to install the Zenhub extension to GitHub to view them properly):
[EPIC] v1.0 starter showcase: https://github.com/gatsbyjs/gatsby/issues/7563
[EPIC] v2.0 starter showcase: https://github.com/gatsbyjs/gatsby/issues/7558